__tests__/comments.test.js

```diff
@@ -0,0 +1,49 @@
+const request = require('supertest');
+const app = require('../app.js');
+const connection = require('../db/connection.js');
+
+const seed = require('../db/seeds/seed.js');
+const data = require('../db/data/test-data/index')
+
+beforeEach(() => seed(data));
+afterAll(() => connection.end());
+
+describe("POST /api/articles/:article_id/comments", () => {
+  it('status:201, responds with a comment created on an article', () => {
+    const newComment = {
+      article_id: 2,
```

Comment:
The comment body doesnt need the article_id - you should get that from the params.